WooCommerce/Classes/ViewRelated/Orders/Order Details/OrderDetailsViewController.swift

```diff
@@ -896,6 +908,8 @@ private extension OrderDetailsViewController {
         static let headerContainerInsets = UIEdgeInsets(top: 0, left: 0, bottom: 8, right: 0)
         static let rowHeight = CGFloat(38)
         static let sectionHeight = CGFloat(44)
+        static let maxWidth = CGFloat(450)
```

Comment:
Having experimented with the various iPad sizes, I think we should consider making this 525.

```suggestion
        static let maxWidth = CGFloat(525)
```

450 results in narrow margins (that seem pointless) on iPad, iPad Air, and iPad Pro 11" in the following configurations:
- Portrait full screen
- Portrait 2/3 screen
- Landscape 2/3 screen

iPad Mini works well with 450, or 525.
Large iPhones in landscape work better with 450, but it's marginal and they seem mostly fine.

I think if you changed the trailing, leading, and centre constraints to be relative to the safe area, not the superview, that would improve large phones.
![large phone superview constraints](https://github.com/woocommerce/woocommerce-ios/assets/2472348/d48f63d8-c77a-4a48-a944-46357e39fce5)

Reply:
> I think we should consider making this 525.

That sounds good to me, initially I experimented with 540-560 when attempting to normalize the designs for the initial proportional approach, since 450 seemed unnecessarily narrow. Changed to `525` here: https://github.com/woocommerce/woocommerce-ios/pull/11714/commits/88a024d3ad9fc5949b5efb46cdf9a069e27affcd

> I think if you changed the trailing, leading, and centre constraints to be relative to the safe area, not the superview, that would improve large phones.

Definitely looks better since closes the gap between order list and order details, and the dynamic island doesn't get in the way of the UI. Changed on https://github.com/woocommerce/woocommerce-ios/pull/11714/commits/dc5056f07d48f9a864ed86868bbe052431da15bf

| Constrained to superview | Constrained to safe area |
|--------|--------|
| ![lhs dynamic island constrained superview](https://github.com/woocommerce/woocommerce-ios/assets/3812076/0775a8b1-e71d-4506-af77-6d06ffa8a8db) | ![lhs dynamic island constrained safearea](https://github.com/woocommerce/woocommerce-ios/assets/3812076/f9c756f0-ae5d-45ee-98d5-0bc3579c3ebc)|
| ![rhs dynamic island constrained superview](https://github.com/woocommerce/woocommerce-ios/assets/3812076/e24a7afa-ee47-467b-a088-cdfa0347d788) | ![rhs dynamic island constrained safearea](https://github.com/woocommerce/woocommerce-ios/assets/3812076/da96a3cd-ba0c-45c6-8d15-2648e98299f7) |

Reply:
Thanks! 

I really don't like that safe area opposite the Dynamic Island in landscape. Maybe it's because of the rounded corners, but they don't affect us much, with the toolbar and tab bar in place. I can't see any other reason for it to be there. 

Never mind...